contracts/DataModel.sol

```diff
@@ -40,6 +42,11 @@ library FrontDoorStructs {
         Job job;
     }
 
+    struct Company{
+        address wallet;
+        uint256 jobsCreated;
```

Comment:
Why do we need an incrementor variable. Is it for future use? or does it have a function already now ??? 👍

Reply:
It works as a counter; that a company has created X number of jobs on our platform. We may also keep track of a company-specific jobs created (for example, an array of job-ids created by a company).

We can remove it if you think it isn't needed.